### ISSUE TITLE
returning string rather than object??

### ISSUE THREAD

**Issue by btmurrell** [ritz078/transform-css-to-js#5]
Your doc says `cssToJS` will return an object. In my usage, it returns a string.  and the string is not a JSON string, so i cannot parse it.  Am I doing something wrong?  

this is the string value i am getting:

```
"{
  RootElement: {
    eWhite: '#FFFFFF',
    eBlack100: '#3c4d5c',
    eBlack200: '#414c56',
    eBlack300: '#364756',
    eBlack400: '#304150',
    eBlack500: '#304250',
    }
}"
```

**Comment by TechStudent10** [comment]
Might be worth a try to use

```js
JSON.parse(data)
```

**Comment by btmurrell** [comment]
> Might be worth a try to use
> 
> ```js
> JSON.parse(data)
> ```

nope. look at the string. that is not valid JSON.  That's my point.

**Comment by TechStudent10** [comment]
> > Might be worth a try to use
> > ```js
> > JSON.parse(data)
> > ```
> 
> nope. look at the string. that is not valid JSON. That's my point.

Ahhhhhh, I don't know who to fix this sorry.

**Comment by i-lq** [comment]
Solved this problem this way, it's a quick fix, I guess.

`json= eval("(()=>{return "+cssToJS(css)+"})()");`
